developer-docs-site/docs/guides/move-guides/index.md

```diff
@@ -42,3 +44,6 @@ Use these external resources to learn about the core Move programming language.
 * [Move Tutorial](https://github.com/move-language/move/tree/main/language/documentation/tutorial)
 * [Move language repository](https://github.com/move-language/move)
 * [Move by example](https://move-book.com/)
+* [Awesome Move resources](https://github.com/MystenLabs/awesome-move)
```

Comment:
Remove this line please.  These examples are not transferrable across versions of Move, and provide a lot of confusion to people.  We should have our own for Aptos Move resources, because I've spent a lot of time explaining to people XYZ doesn't exist.

Reply:
@wrwg says Awesome Move is the canonical home for external resources on the Move programming language. Are you sure you want to dissuade our users from knowing about it?

Reply:
Yes, I do think we should.  I don't believe it is a canonical home for external resources as I've watched countless people confused by it on the Move programming language, and if it was it should be on the Move language organization instead.

Reply:
But I concede, if @wrwg says it's the thing to put, let's put it in.